ledger-core/v2/virtual/integration/stateunavailable_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package small
+
+import (
+	"testing"
+
+	"github.com/ThreeDotsLabs/watermill/message"
+	"github.com/stretchr/testify/require"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/payload"
+	"github.com/insolar/assured-ledger/ledger-core/v2/instrumentation/inslogger"
+	"github.com/insolar/assured-ledger/ledger-core/v2/reference"
+	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/integration/utils"
+	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/statemachine"
+)
+
+func makeVStateUnavailableEvent(t *testing.T, ref insolar.Reference) *statemachine.DispatcherMessage {
+	payLoadMeta := &payload.VStateUnavailable{
+		Reference: ref,
+		Reason:    payload.Inactive,
+	}
+	return makeDispatcherMessage(t, payLoadMeta)
+}
+
+func TestVirtual_VStateUnavailable_NoSuchObject(t *testing.T) {
```

Comment:
I'd suggest adding two more tests for all possible "Reasons", or is it out of scope of that task?

Reply:
I will add